stages/org.osbuild.systemd.unit

```diff
@@ -22,15 +22,28 @@ def main(tree, options):
         unit_dropins_dir = f"{tree}/usr/lib/systemd/user/{unit}.d"
     os.makedirs(unit_dropins_dir, exist_ok=True)
 
-    config = configparser.ConfigParser()
+    # We trick configparser into letting us write multiple instances of the same option by writing them as keys with no
+    # value, so we enable allow_no_value
+    config = configparser.ConfigParser(allow_no_value=True)
     # prevent conversion of the option name to lowercase
     config.optionxform = lambda option: option
 
     for section, opts in cfg.items():
         if not config.has_section(section):
             config.add_section(section)
         for option, value in opts.items():
-            config.set(section, option, str(value))
+            if isinstance(value, list):
+                for v in value:
+                    if option == "Environment":
+                        # Environment arrays are objects
+                        print("env", v)
```

Comment:
Is this intentional or a leftover debug print?

Reply:
Leftover.  We didn't have the `allow_no_value` option set here and it was failing and was driving me a bit crazy :D

Reply:
Fixed.  Thanks!